internal/sdkv2provider/resource_user_identity.go

```diff
@@ -45,14 +46,29 @@ func resourceUserIdentity() *schema.Resource {
 					},
 				}
 			},
+			IdentityUpgraders: []schema.IdentityUpgrader{
+				{
+					Version: 1,
+					Type: tftypes.Object{
+						AttributeTypes: map[string]tftypes.Type{
+							"email": tftypes.String,
+						},
+					},
+				},
+			},
```

Comment:
nit: I think you ended up deleting this when we were looking at this today? 👀

Reply:
whoops, committed, 👍